src/components/FlowChart/FlowSelection.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from "react";
+import { Dropdown } from "react-bootstrap";
+
+function SelectionIndicator({ selectedId, itemId }) {
+  return selectedId === itemId ? (
+    <i className="material-icons text-primary">radio_button_checked</i>
+  ) : (
+    <i className="material-icons">radio_button_unchecked</i>
+  );
+}
+
+function ConnectionIndicator({ connected, show }) {
+  if (!show) return null;
+  return connected ? (
+    <i className="material-icons ml-2 text-success">wifi</i>
+  ) : (
+    <i className="material-icons ml-2 text-warning">wifi_off</i>
+  );
+}
+
+function TitleConnectionIndicator({ connected, show }) {
+  if (!show) return null;
+  return connected ? (
+    <i className="material-icons text-white ml-1 mr-2">wifi</i>
+  ) : (
+    <i className="material-icons text-warning ml-1 mr-2">wifi_off</i>
+  );
+}
+
+function DeleteFlowButton({ show, deleteFlow }) {
+  if (!show) return null;
+  return (
+    <i className="material-icons text-danger float-right" onClick={deleteFlow}>
+      delete
+    </i>
+  );
+}
+
+function EditFlowsButton({ isEditing, onClick }) {
+  return isEditing ? (
+    <i
+      onClick={onClick}
+      className="material-icons float-right cursor-pointer text-success"
+    >
+      done
+    </i>
+  ) : (
+    <i onClick={onClick} className="material-icons float-right cursor-pointer">
+      edit
+    </i>
+  );
+}
+
+export default function FlowSelection({
+  flowOptions,
+  loadFlow,
+  selectedFlowId,
+  createNewFlow,
+  deleteFlow,
+  connected,
+}) {
+  const [isEditing, setIsEditing] = useState(false);
+
+  const toggleEditing = () => {
+    setIsEditing((currentIsEditing) => !currentIsEditing);
+  };
+
+  const userFlows = Object.entries(flowOptions).filter(
+    ([id, flow]) => flow.type !== "example"
+  );
+
+  const exampleFlows = Object.entries(flowOptions).filter(
+    ([id, flow]) => flow.type === "example"
+  );
+
+  const currentFlow = flowOptions[selectedFlowId];
```

Comment:
If the component is about the current flow, we should not pass  `selectedFlowId` and  `flowOptions` to get the currentFlow inside the component but only pass one item, which would be the selectedFlow.  At some point we might start pulling things out of the store but if everything is in the store right now, the selectedFlow as well the flowOptions, there should be a getter of getting the currentFlow from the store.
Some thing, I also didn't get imediatelly:
What are flowOptions? The getter in the store `Store.getFlowOptions()` returns just` _flows`. We call it flowOptions but one flowOption selected by Id, results in a flow.